modules/llvm-project/17.0.3.bcr.1/patches/0003-Add-MODULE.bazel.patch

```diff
@@ -0,0 +1 @@
+../../17.0.3/patches/0003-Add-MODULE.bazel.patch
```

Comment:
This patch needs to be modified to use the new version

Reply:
oops!